### PR TITLE
Build: Node 12, Attempt 4

### PR DESCRIPTION
**Pull request by blowery**
Reverts Automattic/wp-calypso#37607

Fourth shot at moving to node@12, the new LTS.

The github generated revert was https://github.com/Automattic/wp-calypso/pull/37621, but I recreated it under a reasonable branch name.

### CODE REVIEW

**Comment by sirreal** [comment]
🤞

**Comment by sirreal** [comment]
Now time for [https://nodejs.org/en/blog/release/v12.13.1/](`12.13.1`).

**Comment by blowery** [comment]
The logs we got indicated the failure was inside a git-based-module install and the only git hosted module we have anymore is `phone`. https://github.com/Automattic/wp-calypso/pull/27902 moves it to normal registry dependency...

**Comment by sirreal** [comment]
🎉 Go 👏 @blowery 👏 go! 🎉

File: test/e2e/.chromedriver_version [hunk 1]
```diff
@@ -1 +1 @@
-77.0.3865.40
+79.0.3945.36
```
Comment (stephanethomas) [inline test/e2e/.chromedriver_version]: @blowery: I think we should also update the `chromedriver` version number in `test/e2e/package.json` (similarly to what was done in https://github.com/Automattic/wp-calypso/pull/36978). I'm indeed getting the following errors [with E2E](https://circleci.com/gh/Automattic/wp-e2e-tests-for-branches/39498) tests for https://github.com/Automattic/wp-calypso/pull/38333:

```
session not created: This version of ChromeDriver only supports Chrome version 77
```

Comment (blowery) [reply test/e2e/.chromedriver_version]: Thanks! https://github.com/Automattic/wp-calypso/pull/38508

Comment (blowery) [reply test/e2e/.chromedriver_version]: Fixed on master.

Comment (stephanethomas) [reply test/e2e/.chromedriver_version]: Thanks Ben!

**Pull request merged**